### DISCUSSION TITLE
Go suck my dick and eat my shit

### DISCUSSION THREAD

**Discussion by NickxDxx** [ultrafunkamsterdam/undetected-chromedriver#1411] [General]
This project suck as hell! Go eat my shit! :)

**Comment by leesanghyone** [comment]
This project is good, 
Do we pay to use it? I rather think that being able to use it for free is in itself a reason to be grateful.

**Comment by jewqrkqwrwq** [comment]
angry cuz u now need to find ur own solutions to cf rather than downloading a library from github. Ugh poor you i cry for u :/
little baby
